controllers/grid/users/chapter/form/ChapterForm.inc.php

```diff
@@ -195,11 +208,15 @@ public function fetch($request, $template = null, $display = false)
             }
         }
 
+        $selectedChapterAuthorsArray = [];
```

Comment:
Could this (and related) code be moved into `if ($this->getChapter())` above? I think that might make it a little more readable.

Reply:
Yes, good point